### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by nicboul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ This brief tutorial shows **where some of the most used and quoted sysctl/networ
 1. Driver will `schedule a NAPI`, clear the `hard IRQ` and return
 1. Driver raise a `soft IRQ (NET_RX_SOFTIRQ)`
 1. NAPI will poll data from the receive ring buffer until `netdev_budget_usecs` timeout or `netdev_budget` and `dev_weight` packets
-1. Linux will also allocated memory to `sk_buff`
+1. Linux will also allocates memory to `sk_buff`
```
Comment (robbyoconnor) [inline README.md]: Still incorrect grammar: 

Correct would be: 

Linux will also **allocate** memory to `sk_buff`.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Still incorrect grammar.

File: README.md (hunk 1 above)
Comment (robbyoconnor) [reply README.md]: `s/allocates/allocate`

Comment (nicboul) [reply README.md]: true that, fixed !

**Review by leandromoreira — APPROVED** [review]

**Pull request merged**